sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/src/Microsoft.Azure.WebJobs.Extensions.EventGrid.csproj

```diff
@@ -14,6 +14,7 @@
 		<PackageReference Include="Microsoft.Azure.WebJobs" />
 		<PackageReference Include="Microsoft.Extensions.Azure" />
     <PackageReference Include="Azure.Messaging.EventGrid" />
+		<PackageReference Include="Azure.Core" />
```

Comment:
We should be able to remove this and the CloudNativeCloudEvents reference once Azure.Messaging.EventGrid is released with Azure.Core 1.34+

Reply:
Tracking: https://github.com/Azure/azure-sdk-for-net/issues/37628